client.go

```diff
@@ -491,7 +481,7 @@ func (c *Client) sendHub() {
 				// And get rid of the timeout workaround.
 				c.LocalKite.Log.Error("error sending %q: %s", msg, err)
 
-				if err == sockjsclient.ErrSessionClosed {
+				if err == sockjsclient.ErrSessionClosed || strings.Contains(err.Error(), errClosing) {
```

Comment:
> most probably we could get a syscal error like "Connection reset by peer"

good point, I've removed the check - it could be replaced with sth like:

```go
if e, ok := err.(net.Error); ok && !e.Temporary() {
    return
}
```

But I'd need to write a test first to reproduce it. So I'll leave it for one of next PRs.